# 3.0 - Example Observable Multiple Nexts

Demonstrates multiple nexts.  What happens if there is an error call _before_ 
the third timeout finishes?
